### PR TITLE
Eye nfs

### PR DESCRIPTION
**Pull request by RogerRussel**
Make NFS monitoring eye to Beholder.
It is missing some tests, I did it handly, but for auto test I could not, because some permissions problems on Docker, but I test it!

### CODE REVIEW

File: src/Eyes/Nfs/NfsStatus.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+namespace BeholderWebClient\Eyes\Nfs;
+
+use BeholderWebClient\Eyes\Status;
+
+class NfsStatus extends Status {
+
+  const REQUERIMENT_FAIL = 'REQUERIMENT FAIL: stat command not found with type -P stat';
+  const REQUERIMENT_FAIL_NUMBER = 500;
+
+  const PATH_NOT_EXIST = 'Path doesn\'t not exist: ';
```
Comment (alanyoshida) [inline src/Eyes/Nfs/NfsStatus.php]: Doesn't = Does + not
Doesn't Not = double negation ?

File: src/Eyes/Db/DbStatus.php [hunk 2]
```diff
@@ -6,6 +6,9 @@
 
 class DbStatus extends Status {
 
+  const NO_DRIVER = 'No Mysql driver found, it tried use PDO, mysqli and mysql, but neither are they found.';
```
Comment (alanyoshida) [inline src/Eyes/Db/DbStatus.php]: but neither are they found.
but neither of them was found.

File: src/Eyes/Db/DbStatus.php [hunk 3]
```diff
@@ -6,6 +6,9 @@
...
+  const NO_DRIVER_NUMBER = 500; // Because it was a system error
+
   const COULD_NOT_CONNECT_AT_SGBD = 'Could not connect at SGBD';
```
Comment (alanyoshida) [inline src/Eyes/Db/DbStatus.php]: Could not connect at SGBD
Could not connect **to** SGBD

File: src/Eyes/Nfs/NfsStatus.php (hunk 1 above)
Comment (alanyoshida) [inline src/Eyes/Nfs/NfsStatus.php]: Doesn't = Does + not
Doesn't Not = double negation ?

Comment (RogerRussel) [inline src/Eyes/Nfs/NfsStatus.php]: Dump me, sorry :P

Comment (alanyoshida) [inline src/Eyes/Nfs/NfsStatus.php]: Doesn't = Does + not
Doesn't Not = double negation ?

**Comment by RogerRussel** [comment]
I will correct into this merge request, only the problems with NFS, the problems with DB/MYsql problem will be correct into another branch, because they are not made into this branch, and already into master branch.

File: src/Eyes/Db/DbStatus.php (hunk 2 above)
Comment (RogerRussel) [inline src/Eyes/Db/DbStatus.php]: This is the right : "but neither has been found"
https://forum.wordreference.com/threads/neither-of-them-have-has-been-found.2208033/

**Comment by RogerRussel** [comment]
Correct what I should corret into master, and what I should correct into this branch. please look again.

File: doc/PATTERN.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+Like this:
+
+# Beholder Eyes APT
+
+This beholder eyes keep an eye into APT system, APT is just a ficticius name.
+
+## Example of use
+
+Fallow um example of how use it
```
Comment (alanyoshida) [inline doc/PATTERN.md]: Fallow um example of how use it
Follow a example of how to use it

File: doc/nfs/NFS.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+This beholder eye keep an eye into NFS, if it was mounted, if is possible read, write and delete files into it.
+
+## Exemple of use
+
+<code>
+require 'vendor/autoload.php';
+
+$conf = [
+  'eyes' => [
+      'Nfs' => [ // This Nfs is juts an alias, it can be anything
```
Comment (alanyoshida) [inline doc/nfs/NFS.md]: This Nfs is **juts** an alias, it can be anything
This Nfs is **just** an alias, it can be anything

File: doc/nfs/NFS.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# Beholder Eye - NFS
+
+This beholder eye keep an eye into NFS, if it was mounted, if is possible read, write and delete files into it.
```
Comment (alanyoshida) [inline doc/nfs/NFS.md]: if is possible read
if **it's** possible **to** read

**Pull request merged**